### PR TITLE
Update us.m3u [ABC, CBS, FOX, NBC, Other US Sport Channels]

### PR DESCRIPTION
**Pull request by Bobcool101**
This patch has a bunch of different regional ABC, CBS, FOX, NBC Networks. It also includes NBA Gamepass.

### CODE REVIEW

**Review by its0x08 — DISMISSED** [review]

File: streams/us.m3u [hunk 1]
```diff
@@ -1760,3 +1760,255 @@ https://wtvjlive-f.akamaihd.net/i/MIAMI_STREAM1@19309/master.m3u8
...
+#EXTINF:-1 tvg-id="KTXLDT1.us" status="",FOX Sacramento CA (720p)
+http://51.161.118.146:8080/ISG03_FOX_SACRAMENTO_CA_KTXL/video.m3u8
+#EXTINF:-1 tvg-id="KSWBDT1.us" status="",FOX San Diego CA (720p)
+http://51.161.118.146:8080/ISG03_FOX_SAN_DIEGO_CA_KSWB/video.m3u8
+#EXTINF:-1 tvg-id="KCPQDT1.us" status="",FOX Seattle WA (720p)
+http://51.161.118.146:8080/ISG03_FOX_SEATTLE_WA_KCPQ/video.m3u8
+#EXTINF:-1 tvg-id="WTTGDT1.us" status="",FOX Washington DC (720p)
+http://51.161.118.146:8080/ISG03_FOX_WASHINGTON_DC_WTTG/video.m3u8
+#EXTINF:-1 tvg-id="FS1.us" status="",FS1 (720p)
+http://104.238.221.56:8080/live/823868279/9507295574/6014.m3u8?token=Q0dRVxcMEAMbCFkFVABTAVAAVAUBVlUMUAdSVgcDDFtaVl1UB1IDDgEbTRBDQBdXVF5vCFBACQ4CBFAbQERGXEo6UFJGXhAIDApRBggWSRBBWl0EFlgDFBBFDVNACxcLDFcKAkZIEFxBTQRAXlUJbVFWRghXBxECXEAIW04TWldnAVxbBQpWGwMbUBAcFgxBRREKD0EOXxQQVw1DEFBBXBpfGwVQXAIbFRsCXUVaEUBMEQpDYTERFBBQHEMHX0ZQVwsbDEYJAUwBG00QWUc6QFBARBNRA15dQBdeFVITGRtVBEFpBwtcV1xaFVtfWhYQDxEFQxhAXldcXBBYEG5FUFxHAxRdVwMOG0Q=
```
Comment (Central302) [inline streams/us.m3u:1912]: FOX Sports Channels Are Not Allowed (Blacklisted Due to DMCA, See [Below](https://github.com/iptv-org/database/blob/master/data/blocklist.csv)

File: streams/us.m3u [hunk 2]
```diff
@@ -1760,3 +1760,255 @@ https://wtvjlive-f.akamaihd.net/i/MIAMI_STREAM1@19309/master.m3u8
...
+#EXTINF:-1 tvg-id="MSG.us" status="",MSG (1080p)
+https://msggo-live.net/v1/manifest/1a954fea9a9e371ab24e8226b756520725bb3f2d/MSGN-LIVE-001/ef513621-4497-4898-9a49-89d812b10faa/0.m3u8
```
Comment (Central302) [inline streams/us.m3u:1914]: MSG, But Tokened and USA IP ONLY.

**Review by Central302 — CHANGES_REQUESTED** [review]

**Comment by Bobcool101** [comment]
Changes requested by Central302 have been added

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
I have 404s on the https://d17f1mg1q6g54e.cloudfront.net/ links

**Comment by Bobcool101** [comment]
> I have 404s on the https://d17f1mg1q6g54e.cloudfront.net/ links

Its because that team is not currently playing a game, wait until a team starts playing and you'll be able to watch. I linked a 24/7 nbatv stream in there to show it works: https://d17f1mg1q6g54e.cloudfront.net/hls/nbatv/chunks.m3u8

**Comment by LaneSh4d0w** [comment]
OK.

**Review by BellezaEmporium — APPROVED** [review]

**Review by what-a — APPROVED** [review]

**Review by Central302 — APPROVED** [review]

**Pull request merged**